client/src/api/behavior.js

```diff
@@ -0,0 +1,9 @@
+export async function addBehavior(studentId, behaviorName) {
+    const res = await fetch("behavior/add/" + studentId, {
+        method: "GET",
```

Comment:
if we're "adding a behavior" shouldn't the method be PUT or POST (probably POST?)?

Reply:
Oh good catch! Backend is POST so I fixed it to POST ~~Thanks!